bot/main.py

```diff
@@ -27,6 +28,16 @@
     logging.basicConfig(level=LOGGING_LEVEL)
 
 
+spam_log_file_clear_interval = 3
+spam_log_file = str(
+    pathlib.Path(
+        os.path.join(
+            os.path.abspath(os.getcwd()), "bot", "cogs", "spam", "spam_log.txt"
+        )
+    ).resolve()
```

Comment:
We define this in two locations using a bunch of string literals. This should be a constant provided from one location.

Reply:
It looks like we can pass `*` , i.e. arguments, to the `client.load_extension` function on line 69.

Might be nice to pass this path and the clear interval (assuming we can handle the clearing of the log from within the cog) 

https://discordpy.readthedocs.io/en/stable/ext/commands/api.html?highlight=load_extension#discord.ext.commands.Bot.load_extension